### DISCUSSION TITLE
Israel_swissGerman

### DISCUSSION THREAD

**Discussion by chagai95** [Estheryu991/SwissGerman_Dictionary#30] [General]
Lieber Esthi ich komme aus Israel und würde auch gerne Schweizer Deutsch lernen. Ich programmiere seit 5 Jahren

שלום אסתי אני רוצה ללמוד שוויצרית.

**Comment by esthicodes** [comment]
- אני אוהב לתכנת
- אני אוהב את אמא שלי
- אני אוהב קוקה קולה

**Comment by esthicodes** [comment]
על הדבש ועל העוקץ
לכבוד חג האסיף מסכם שר החקלאות עודד פורר קדנציה קצרה אך סוערת

**Reply by esthicodes** [reply]
![image](https://user-images.githubusercontent.com/78131082/196703010-030590b6-c545-4fd7-b2c8-ecf6b7863b77.jpeg)

**Reply by esthicodes** [reply]
![image](https://user-images.githubusercontent.com/78131082/196703164-45e1d9b4-f203-4000-92df-6320d5dca005.jpeg)

**Reply by esthicodes** [reply]
> על הדבש ועל העוקץ
> לכבוד חג האסיף מסכם שר החקלאות עודד פורר קדנציה קצרה אך סוערת


es isch ein Satzt von ei Lied

mein Hönig

**Comment by esthicodes** [comment]
הגעת ליעד

- הגעת ליעד

https://user-images.githubusercontent.com/78131082/196705124-14863469-ec2c-45fc-9c2a-197ba575fba7.MOV

**Comment by esthicodes** [comment]
- מחיר הגמילה
- מחיר הגמילה
the cost oft rehub

**Comment by esthicodes** [comment]
- Du hasch dini Ziel erreicht

**Comment by esthicodes** [comment]
תודה רבה
 
Danke
